rllib/core/models/configs.py

```diff
@@ -57,27 +111,49 @@ class _MLPConfig(ModelConfig):
 
     Attributes:
         input_dims: A 1D tensor indicating the input dimension, e.g. `[32]`.
-        hidden_layer_dims: The sizes of the hidden layers. If an empty list, only a
-            single layer will be built of size `output_dims[0]`.
+        hidden_layer_dims: The sizes of the hidden layers. If an empty list,
+            `output_layer_dim` must be provided (int) and only a single layer will be
+            built.
         hidden_layer_activation: The activation function to use after each layer (
             except for the output).
         hidden_layer_use_layernorm: Whether to insert a LayerNorm functionality
             in between each hidden layer's output and its activation.
-        output_dims: A 1D Tensor indicating the size of the output layer. This may be
+        hidden_layer_use_bias: Whether to use bias on all dense layers in the network
+            (excluding a possible separate output layer defined by `output_layer_dim`).
+        output_layer_dim: An int indicating the size of the output layer. This may be
```

Comment:
This is a general concept throughout this PR:
`output_layer_dim`: User can set this to configure a last dense layer that may (or may not) abide to different activations/bias/layernorm settings, etc..)
`output_dims`: A property that get calculated automatically by the config object. Catalogs can use this property to conveniently stack together different model components (encoder + encoder OR encoder + head).